docs/contribute/general/contribution_attribution.rst

```diff
@@ -0,0 +1,70 @@
+..
+   # *******************************************************************************
+   # Copyright (c) 2025 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+
+.. _contribution_attribution:
+
+Actions to Ensure Proper Contribution Attribution in Eclipse SCORE
+=====================================================================
+
+To ensure that contributions are correctly attributed to organizations in the **Eclipse SCORE** project, follow these steps:
+
+Actions for Contributors
+--------------------------
+
+1. **Create an Eclipse Foundation Account**
+   - If you don't have one, register at `Eclipse Foundation <https://accounts.eclipse.org/user/register>`_.
```

Comment:
```suggestion
   - If you don't have one, register at `Eclipse Foundation <https://accounts.eclipse.org/user/register>`_.
   - Use your corporate email account (the one you will be using for commits).
```

Reply:
I guess the question here is whether this is a personal account or a company account.
i.e. what happens to the account when someone changes employers? Do they change the email address and all contributions are changed? 😆

Reply:
It's definitely a personal account. But I guess we use the employers email, and follow some process later on to change it.

> [If I change employers, what happens to my committer status at Eclipse?](https://www.eclipse.org/legal/faq/#h.tq9t96dkfm2y)
Your committer status at the Eclipse Foundation is not based on your employment status.